lib/response.js

```diff
@@ -229,7 +233,16 @@ res.send = function send(body) {
     this.end();
   } else {
     // respond
-    this.end(chunk, encoding);
+    if (isChunkBlob) {
+      var response = this;
+      chunk.arrayBuffer().then(function (result) {
+        response.end(Buffer.from(result), encoding);
```

Comment:
I don't think you should be passing the `encoding` argument here since you are passing a `Buffer` object that has already been encoded -- it is just for string arguments afaik

Reply:
yes, just remove this thing from here